package.json

```diff
@@ -36,6 +37,7 @@
     "eslint": "8.27.0",
     "eslint-config-next": "13.0.3",
     "eslint-plugin-storybook": "^0.6.7",
+    "file-loader": "^6.2.0",
```

Comment:
Why do you need this loader? And does it work without adding it to the next.config.js?

Reply:
I should have add it to the next.config.js - but I did not use it yet, therefore I'm gonna wrap a comment on it.